devs/docs/basics.rst

```diff
@@ -101,6 +101,17 @@ directory.
 
 Here, ``<PROJECT_ROOT>`` is the root of your Git repository.
 
+Checkstyle
+----------
+
+The Checkstyle plugin enforces rules defined in `/gradle/checkstyle/rules.xml`.
```

Comment:
`<PROJECT_ROOT>/gradle/...` (link in L102)

Reply:
I think that's a bit redundant because `/` already indicates the root. I'll add it to avoid confusion.

Reply:
no strong opinion, tbh, only that it should be consistent throughout the docs :)